cmd/bolt-agent/main.go

```diff
@@ -593,7 +606,42 @@ func nodeDataChecker(cmdCtx *cli.Context) error {
 		}
 
 		glog.Info("Waiting for events...")
-		utils.WaitAll(nodeDataChecker)
+
+		g.Go(func() error {
```

Comment:
👍

Reply:
it still needs some refactoring in the near future. EventLoop should take context (gctx) as parameter and not constructor.